### PR TITLE
Search operation on operation-sidebar is now working

### PR DESCRIPTION
**Pull request by yuricarvalhop**

### CODE REVIEW

File: app/components/workflows/operations-sidebar.js [hunk 1]
```diff
@@ -2,15 +2,43 @@ import Ember from 'ember';
 import Ps from 'npm:perfect-scrollbar';
 
 export default Ember.Component.extend({
+
+  filterText: '',
+  store: Ember.inject.service('store'),
+
+  init() {
+    this._super(...arguments);
+    this.addObserver('filterText', this, 'filterDidChange');
```
Comment (michelboaventura) [inline app/components/workflows/operations-sidebar.js]: There is a better way to do this. You can use a computed property so you don't need to add this observer and implement the function below: https://guides.emberjs.com/v2.14.0/object-model/computed-properties/

File: package.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "ember-tether": "^1.0.0-beta.0",
+    "metismenu": "^2.7.0",
```
Comment (michelboaventura) [inline package.json]: You have added metismenu here but it is already on vendor/metisMenu. I think this can gone.

File: vendor/metisMenu/metisMenu.js [hunk 3]
```diff
@@ -1,375 +1,354 @@
...
-      exports: {}
-    };
-    factory(global.jquery);
-    global.metisMenu = mod.exports;
-  }
-})(this, function (_jquery) {
-  'use strict';
-
-  var _jquery2 = _interopRequireDefault(_jquery);
+const Util = (($) => {
```
Comment (michelboaventura) [inline vendor/metisMenu/metisMenu.js]: Why did you change metis menu code here?

File: app/components/workflows/operations-sidebar.js (hunk 1 above)
Comment (yuricarvalhop) [inline app/components/workflows/operations-sidebar.js]: Done.

File: package.json (hunk 2 above)
Comment (yuricarvalhop) [inline package.json]: Fixed.

File: vendor/metisMenu/metisMenu.js (hunk 3 above)
Comment (yuricarvalhop) [inline vendor/metisMenu/metisMenu.js]: The metis menu, that was on vendor was old. It lacked the dispose function among others. I upgraded the metisMenu with npm, but it was not being imported, so I copied the upgraded and complete version to vendor. If you check, the code is completely different, including the  dispose(), that destroys the menu, which i needed, to concatenate between models (Grouped and not grouped).
